samples/src/main/java/com/example/bigquery/RelaxTableQuery.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+// [START bigquery_relax_column_query_append]
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Job;
+import com.google.cloud.bigquery.JobId;
+import com.google.cloud.bigquery.JobInfo;
+import com.google.cloud.bigquery.JobInfo.SchemaUpdateOption;
+import com.google.cloud.bigquery.JobInfo.WriteDisposition;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.TableId;
+import com.google.cloud.bigquery.TableResult;
+import com.google.common.collect.ImmutableList;
+import java.util.UUID;
+
+public class RelaxTableQuery {
+
+  public static void runRelaxTableQuery() throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "MY_PROJECT_ID";
+    String datasetName = "MY_DATASET_NAME";
```

Comment:
@kurtisvg, do we have a convention for input fields like:
"MY_PROJECT_ID" is all caps because it is some field they have to already have

Where
"my_dataset_name" is lower case to show this is a user provided field?

Reply:
I think they're both fields to be provided by user - MY_PROJECT_ID is already set only for testing. If they just run the sample, they will still need to provide the projectId.

Reply:
We don't have a convention AFIAK - I prefer `"ALL-CAPS-WITH-HYPENS"` but it's not really a significant difference